app/views/rails_admin/main/index.html.haml

```diff
@@ -39,11 +51,43 @@
   jQuery(function($) {
   #{ordered_filter_string}
   });
+- if horiz_scroll_enabled && horiz_scroll_frozen_cols > 0
+  :javascript
+    (function(){
+      if (window.rails_admin_horizontal_scroll_list) { return; } // Don't add this handler multiple times.
+      window.rails_admin_horizontal_scroll_list = true;
+      var $ = jQuery;
+      var setFrozenColPositions = function(){
+        var firstPosition, $td;
+        $('#bulk_form').find('table tr').each(function(index, tr){
+          $(tr).find('.scroll-frozen').each(function(idx, td){
+            $td = $(td);
+            if (idx === 0) {
+              firstPosition = $td.position().left;
+            }
+            td.style.left = ($td.position().left-firstPosition)+'px';
+          });
+        });
+      };
+      $(window).on('load', setFrozenColPositions); // Update after link icons load.
+      $(document).on('rails_admin.dom_ready', setFrozenColPositions);
+    }());
```

Comment:
Place your javascripts and stylesheets in appropriate location, not putting them into the view.